src/DynamoDbModel.php

```diff
@@ -460,4 +460,19 @@ public function __wakeup()
         parent::__wakeup();
         $this->setupDynamoDb();
     }
+
+    /**
+     * Returns approximate count of items in DynamoDB table for current model
+     * This item count is updated every six hours by AWS
+     *
+     * @return int
+     */
+    public static function getItemCount()
+    {
+        $model = new static;
+        $describeTable = $model->getClient()->describeTable([
+            'TableName' => $model->getTable()
+        ]);
+        return $describeTable->get('Table')['ItemCount'];
```

Comment:
I prefer that the query is actually done in the DynamoDbQueryBuilder. And that would allow us to do:

```
return $model->newQuery()->getTableItemCount();
```